### ISSUE TITLE
Pytorch implementation

### ISSUE THREAD

**Issue by zhaoxin94** [RockySJ/WDGRL#7]
Hi, I'm trying to implement your work with the Pytorch framework. But I can't get good accuracy. 
The training code I use is mainly from  [jvanvugt/pytorch-domain-adaptation](https://github.com/jvanvugt/pytorch-domain-adaptation)
Have you implemented your work with Pytorch? Or  Could you give me some advice about my question?

Many thanks！

**Comment by Drenches** [comment]
+1

**Comment by zhiqihuang** [comment]
+1

**Comment by ZhongLIFR** [comment]
+1
